src/index/ivf_raft/ivf_raft.cu

```diff
@@ -14,36 +14,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 #include "ivf_raft.cuh"
 #include "knowhere/factory.h"
 #include "knowhere/index_node_thread_pool_wrapper.h"
 
-constexpr uint32_t cuda_concurrent_size = 16;
-
 namespace knowhere {
 
-static std::shared_ptr<ThreadPool>
-GlobalThreadPoolRaft() {
-    static std::shared_ptr<ThreadPool> pool = std::make_shared<ThreadPool>(cuda_concurrent_size);
-    return pool;
-}
 KNOWHERE_REGISTER_GLOBAL(GPU_RAFT_IVF_FLAT, [](const Object& object) {
     return Index<IndexNodeThreadPoolWrapper>::Create(
-        std::make_unique<RaftIvfIndexNode<detail::raft_ivf_flat_index>>(object), GlobalThreadPoolRaft());
```

Comment:
Why we removed all thread pools for RAFT?

Reply:
Because of the conflict between raft and folly, they can't be on the same compilation unit. So thread pool support for raft is moved to index_node_thread_pool_wrapper.cc